k8s.gcr.io/images/k8s-staging-cluster-api-azure/images.yaml

```diff
@@ -49,6 +49,7 @@
     "sha256:e24334ad9d875c72fe579717f4fcf80010b8f69ab5e2cce4d924e22ce309385d": ["v1.5.3"]
     "sha256:e2b964c28e4ba8fff5cd8340bab62c23e867c411941eb64554c68cf900a572c3": ["v1.1.0"]
     "sha256:e3a22baa51152606d3224f6cadf652ebeb296b0e74d26ba337f6adc6ddb1e21c": ["v1.6.0"]
+    "sha256:e531b11948c14537b53c196ec6c63301d177e1b199262c9115f408b0acc9b642": ["v1.8.0"]
```

Comment:
Confirmed that the corresponding image has the correct tag:

- https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-azure/global/cluster-api-azure-controller@sha256:e531b11948c14537b53c196ec6c63301d177e1b199262c9115f408b0acc9b642/details